apps/wallet/src/ui/app/ApiProvider.ts

```diff
@@ -71,15 +79,47 @@ function getDefaultAPI(env: API_ENV) {
 
 export const DEFAULT_API_ENV = getDefaultApiEnv();
 
+type NetworkTypes = keyof typeof API_ENV;
+
+const EXCLUDE_STAGING = process.env.SHOW_STAGING !== 'false' && API_ENV.staging;
+
+export const generateActiveNetworkList = (): NetworkTypes[] => {
+    const excludedNetworks: NetworkTypes[] = [];
+    if (EXCLUDE_STAGING) {
+        excludedNetworks.push(API_ENV.staging);
+    }
+
+    // Deal with edge case where user has customRPC or testnet is disabled
+    const excludeCustomRPC =
```

Comment:
I don't love this `&&` pattern, I'd just keep these as booleans and push in the `API_ENV` value in the if body.

Reply:
done.